apps/user-profile/src/components/modules/OnboardingLandingPage/OnboardingLandingPage.js

```diff
@@ -1,272 +1,385 @@
+              <CheckboxField
+                customLabel={
+                  <Typography type="pSmall">
+                    I have read and agree to the <a>Terms and Conditions</a>
+                  </Typography>
+                }
+                onChange={onTermsAndConditionChange}
+                checked={person.termsAndConditions}
+                required={true}
+              />
+              {/* <CheckboxField
                                 customLabel={<Typography type="pSmall">Join our weekly Newsletter </Typography>}
                                 onChange={onJoinNewsLetterChange}
                                 required={true}
                             /> */}
-                            <ButtonContainer>
-                                <Button
-                                    buttonType="alternative"
-                                    buttonSize="xl"
-                                    onClick={onCancelClick}
-                                >
-                                    Cancel
-                                </Button>
-                                <Button
-                                    type='submit'
-                                    buttonType="secondary"
-                                    buttonSize="xl"
-                                    onClick={onContinueClick}
-                                    disabled={!formValidation.isFormValid()}
-                                >
-                                    Save and Continue
-                                </Button>
-                            </ButtonContainer>
-                        </FormFooter>
-                    </FormContainer>
-                </OnboardingContainer>
-            </PageContainer>
-        </>)
+              <ButtonContainer>
+                <Button
+                  buttonType="alternative"
+                  buttonSize="xl"
+                  onClick={onCancelClick}
+                >
+                  Cancel
+                </Button>
+                <Button
+                  type="submit"
+                  buttonType="secondary"
+                  buttonSize="xl"
+                  onClick={onContinueClick}
+                  disabled={!formValidation.isFormValid()}
+                >
+                  Save and Continue
+                </Button>
+              </ButtonContainer>
+            </FormFooter>
+          </FormContainer>
+        </OnboardingContainer>
+      </PageContainer>
+    </>
+  );
 
-    // return (
-    //     <OnboardingForm>
-    //         <form>
-    //             <FormFieldMargin>
-    //                 <FormField
-    //                     error=""
-    //                     label="FIRST NAME"
-    //                     value={person.firstName}
-    //                     onChange={handleFirstNameChange}
-    //                     placeholder="Placeholder"
-    //                 />
-    //             </FormFieldMargin>
+  // return (
```

Comment:
Why is this commented out?

Reply:
It was there before I started working on this file.